sql/snowflake/SnowflakeLexer.g4

```diff
@@ -343,6 +343,7 @@ FILE_FORMAT:                                           'FILE_FORMAT';
 FILTER:                                                'FILTER';
 FIRST:                                                 'FIRST';
 FIRST_NAME:                                            'FIRST_NAME';
+FIRST_VALUE:                                           'FIRST_VALUE';
```

Comment:
I guess new tokens (`FIRST_VALUE`, `NVL2`, `RESPECT`) should be added to [non_reserved_words](https://github.com/antlr/grammars-v4/blob/master/sql/snowflake/SnowflakeParser.g4#L3357) because they can be identifiers.

Reply:
Yep I forget about that, thank you.